src/operator/pooling.cc

```diff
@@ -85,6 +85,10 @@ DMLC_REGISTER_PARAMETER(PoolingParam);
 MXNET_REGISTER_OP_PROPERTY(Pooling, PoolingProp)
 .describe(R"code(Perform pooling on the input.
 
+The shapes for 1-D pooling are
+- **data**: *(batch_size, channel, width)*,
```

Comment:
also remove the content: `1-D pooling is special case of 2-D pooling with width=1 and kernel[1]=1.`